audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/service/StroomAuditService.java

```diff
@@ -57,8 +61,10 @@
     private static final String SYSTEM_CLASSIFICATION = "systemClassification is {}";
     private static final System SYSTEM = new System();
 
-    protected static final String READ_SUCCESS = "READ_REQUEST_COMPLETED";
-    protected static final String REQUEST_SUCCESS = "REGISTER_REQUEST_COMPLETED";
+    /** read request completed */
+    public static final String READ_SUCCESS = "READ_REQUEST_COMPLETED";
+    /** register request completed */
+    public static final String REQUEST_SUCCESS = "REGISTER_REQUEST_COMPLETED";
```

Comment:
Why are these JavaDoc comments? Can they not be simple in-line comments using `//`?

Reply:
They are public so I get warnings.